Kalugina Daria

kaluginadaria opened the pull request:
All tasks are done + some extra tasks (bugs, security issues)

flacours commented:
Hi, i'm having problem running your program. I have tried to fix stuff but there are too many problems. can you tried to fix my questions
1- the installation instruction are not precise. Which version of python you need ?
2- in main.py os.exit() doesn't exist in python 2 or 3 are you sure it's valid ?
3- environs is missing from the project requirements.txt . can you add the package you need
4- i think you need an environment variable can you indicate in the instruction if i need one.

kaluginadaria commented:
Hi, I added .env file to .gitignore just because it is not safe to store env variables like secret keys in source code, but now .env is in repository so you can run app witthout any problems. I fixed 2nd and 3rd issues and changed installation instruction
now its:
```sh
virtualenv -p python2.7 env 
source env/bin/activate
pip install -r requirements.txt
python main.py initdb
python main.py migrate
python main.py
```

flacours commented:
thank you .
